kyma/extensions/modules/api-gateway-regular.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: operator.kyma-project.io/v1beta2
+kind: ModuleTemplate
```

Comment:
I don't think that these files are currently necessary, because it's only for testing part, and should be in `tests` directory.